src/models/sighting/useDeleteSighting.js

```diff
@@ -30,6 +41,10 @@ export default function useDeleteSighting() {
       return false;
     } catch (postError) {
       setLoading(false);
+      const vulnerableIndividualGuid =
+        postError?.response?.data?.vulnerableIndividualGuid;
+      if (vulnerableIndividualGuid)
+        setVulnerableIndividual(vulnerableIndividualGuid);
```

Comment:
Is there a reason to set this conditionally rather than on every error. If the vulnerableIndividualGuid state has a value and a subsequent request returns an error without a `response?.data?.vulnerableIndividualGuid`, shouldn't the local state be cleared?

Reply:
You're totally right.